Sources/ApolloTestSupport/XCTAssertHelpers.swift

```diff
@@ -153,9 +153,7 @@ public struct XCTFailure: Error, CustomNSError {
 }
 
 public extension Optional {
-  var xctUnwrapped: Wrapped {
-    get throws {
-      try XCTUnwrap(self)
-    }
+  func xctUnwrapped(file: String = #file, line: Int = #line) throws -> Wrapped {
+    try XCTUnwrap(self)
```

Comment:
you actually have to pass the `file` and `line` parameters here for this to work, FYI

Reply:
Hahahah right... I knew that. Lol thanks for noticing. I'll fix it in the next PR.

Reply:
real easy to miss when it doesn't complain about the unused params!